src/API/users.api.json

```diff
@@ -0,0 +1,21 @@
+{
+    "routes":{
+        "getUsers": {
+            "http": {
+                "method": "GET",
+                "request_uri": "/users"
+            },
+            "params": {
+                "page": {
+                    "optional": false
+                },
+                "page_size": {
+                    "optional": false
+                }
+            },
+            "output": {
+                "array":"\\Ionic\\Users\\Models\\User"
```

Comment:
Would a `$client->getLastMeta()` work? If it's only important for pagination requests we could implement a better pagination helper which saves that information but just build it on top of the getLastMeta information.